### PR TITLE
Remane asyncify-side-module to asyncify-relocatable

### PR DESCRIPTION
**Pull request by nokotan**
Related: <https://github.com/emscripten-core/emscripten/pull/15893#discussion_r849517443>

`--pass-arg=asyncify-side-module` option will be used not only from side modules, but also from main modules.

### CODE REVIEW

File: src/passes/Asyncify.cpp [hunk 1]
```diff
@@ -1532,7 +1532,7 @@ struct Asyncify : public Pass {
     auto verbose =
       runner->options.getArgumentOrDefault("asyncify-verbose", "") != "";
     auto sideModule =
```
Comment (kripken) [inline src/passes/Asyncify.cpp:1534]: Please also rename this local and other places it appears to `relocatable`.

**Review by kripken — COMMENTED** [review]
Thanks! This will definitely be better. lgtm % comment.

File: src/passes/Asyncify.cpp (hunk 1 above)
Comment (nokotan) [reply src/passes/Asyncify.cpp]: I'd forgot the local variable name. Thanks!

**Review by kripken — APPROVED** [review]
Perfect!

**Pull request merged**